testplans/lotus-soup/_compositions/baseline-k8s-3-2.toml

```diff
@@ -26,7 +26,7 @@
   miners = "2"
   genesis_timestamp_offset = "0"
   balance = "20000000" # These balances will work for maximum 100 nodes, as TotalFilecoin is 2B
-  sectors = "10"
+  sectors = "4"
```

Comment:
We are moving to 8MiB sectors, so no need to preseal 10 sectors when starting the miners, as it takes unnecessary time.